### ISSUE TITLE
[Adaptive Cards>Blogs]: Flags elements whose role is none or presentation and which cause the role conflict resolution to trigger.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8217]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS Version: 22H2 (Build 25267.1000)
Browser: Edge Dev (110.0.1556.0 (Official build) dev (64-bit)
URL: https://adaptivecards.io/blog/
Tool: Accessibility Insights for Web

**Repro Steps:**
1. Open the above URL in edge dev browser.
2. Run the fast pass of Accessibility Insights for Web
3. Observe the issue.

**Actual Result:**
Flags elements whose role is none or presentation and which cause the role conflict resolution to trigger.
Element is not focusable or Element does not have global ARIA attribute

**Expected Result:**
Element should be focusable or Element should have global ARIA attribute.

**Notes:**
This issue is repro throughout for the 'Blogs' tab.

**User Impact:**
Screen reader user will have difficulty in interacting with the contents and in understanding the purpose and its relationship with the content.

**MAS Reference link:**
https://microsoft.sharepoint.com/:w:/s/accessibility/EXdDUZzldv5MgwDaMHnTzcMBcT8opKGncYjU-gCJ4uqQdg?e=9nuVmt

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

<img width="950" alt="MAS1 3 1 Flags elements whose role is none or presentation and which cause the role conflict resolution to trigger" src="https://user-images.githubusercontent.com/104483005/211264622-7c257338-0e45-478b-bb39-98c46c540d9b.png">
<img width="959" alt="MAS1 3 1 Flags elements whose role is none" src="https://user-images.githubusercontent.com/104483005/211269655-b1347300-9aca-40f6-82f0-f8bb8f8901c5.png">

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/ecf120a9-6fd1-4030-b622-e185d8e6a3b7)
